Documentation/html_viewer/js/dictionary_functions.js

```diff
@@ -25,6 +30,29 @@
 		return tState.data;
 	}
 	
+
+	//customized sort for API dictionaries
+	function compareDictionaryObject(entryObject1,entryObject2) {
+		if(entryObject1["display name"] == "LiveCode Script" && entryObject2["display name"] == "LiveCode Builder")
+			return -1;
+		if(entryObject2["display name"] == "LiveCode Script" && entryObject1["display name"] == "LiveCode Builder")
+			return 1;
+		if(entryObject1["display name"] == "LiveCode Script")
+			return -3;
+		if(entryObject2["display name"] == "LiveCode Script")
+			return 3;
+		if(entryObject1["display name"] == "LiveCode Builder")
+			return -2;
+		if(entryObject2["display name"] == "LiveCode Builder")
+			return 2;
+		if(entryObject1["display name"].toLowerCase() < entryObject2["display name"].toLowerCase())
+			return -1;
+		if (entryObject1["display name"].toLowerCase() > entryObject2["display name"].toLowerCase())
+			return 1;
+		return 0;
+	}
+
```

Comment:
Looks like you are correct.  I thought that I tried that combination when testing before I added the 2 & 3.  I'll verify this evening on my Mac and update the PR.